crates/bindings-macro/src/table.rs

```diff
@@ -526,14 +526,11 @@ pub(crate) fn table_impl(mut args: TableArgs, mut item: MutItem<syn::DeriveInput
     let integrate_gen_col = sequenced_columns.iter().map(|col| {
         let field = col.field.ident.unwrap();
         quote_spanned!(field.span()=>
-            if spacetimedb::table::IsSequenceTrigger::is_sequence_trigger(&_row.#field) {
-                _row.#field = spacetimedb::sats::bsatn::from_reader(_in).unwrap();
-            }
+            spacetimedb::table::SequenceTrigger::maybe_decode_into(&mut __row.#field, &mut __generated_cols);
```

Comment:
No, because `__generated_cols` gets mutated - it's not copying it for every `maybe_decode_into` invocation, which would look like `&mut { __generated_cols }`, the variable is declared as mutable and the pointer into the array gets bumped for every successful decode.